packages/Sortable/src/components/Item/Item.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import HandleIcon from "@paprika/icon/lib/DragHandle";
+import { Draggable } from "react-beautiful-dnd";
+import Button from "@paprika/button";
+import SortableContext from "../../SortableContext";
+import { itemStyles, itemIndexStyles, itemHandleStyles, itemBodyStyles, itemCloseStyles } from "./Item.styles";
+
+const propTypes = {
+  child: PropTypes.node.isRequired,
+  index: PropTypes.number.isRequired,
+};
+
+const Item = ({ child, index }) => {
+  const { hasIndexes, onRemove } = React.useContext(SortableContext);
+
+  function handleRemove() {
+    onRemove(index);
+  }
+
+  return (
+    <Draggable draggableId={`draggable-${child.props["data-drag-id"]}`} index={index}>
+      {(provided, snapshot) => {
+        return (
+          <li
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            css={itemStyles}
+            data-is-dragging={snapshot.isDragging ? true : undefined}
+            isDragging={snapshot.isDragging}
+            ref={provided.innerRef}
```

Comment:
I can't find where the reference for  `ref={provide.innerRef}` is being used can you explain to me a little bit about why we are using it :)

Reply:
> `<Draggable />` and `<Droppable />` components both require a `HTMLElement` to be provided to them. This is done using the `innerRef` property on the `DraggableProvided` and `DroppableProvided` objects.

Source: https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/using-inner-ref.md